src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_api.py

```diff
@@ -1088,12 +1088,12 @@ class PROCESSENTRY32(ctypes.Structure):
     try:
         process_entry = PROCESSENTRY32()
         process_entry.dwSize = ctypes.sizeof(PROCESSENTRY32)
-        if not kernel32.Process32First(snapshot, ctypes.byref(process_entry)):
+        if not kernel32.Process32First(ctypes.c_void_p(snapshot), ctypes.byref(process_entry)):
```

Comment:
The issue with PyPy is that this call failed and thus the ppid was always None (which made `debugpy` misbehave later on).